src/solve.jl

```diff
@@ -43,7 +43,7 @@ If some options are not defined, we may fall back to default values.
 - `options` -- options for solving the problem
 """
 function solve(system::AbstractSystem,
-               options_input::Options)::AbstractReachSolution
+               options_input::Options)::Union{AbstractReachSolution, Bool}
```

Comment:
@mforets: Is this okay or should we create a `AbstractReachSolution` type for the property checking result?